csharp-basics/exercises/FlowOfControl/RealExercise4/Program.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+
+namespace RealExercise4
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            string[] days = { "dummyDay", "Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturtday"};
+            int dayNumber;
+
+            Console.WriteLine("Please input your day number here:");
+            int.TryParse(Console.ReadLine(), out dayNumber);
+
+            if (dayNumber < 6)
```

Comment:
ieliec tad te jau dayNumber < 6 && dayNumber != 0, lai nav 2 reizes jaraksta sis Console.WriteLine("Not a valid day");